litgpt/lora.py

```diff
@@ -122,8 +122,8 @@ def __init__(
 
         # Actual trainable parameters
         if r > 0:
-            self.lora_A = nn.Parameter(torch.zeros((r, in_features)))
-            self.lora_B = nn.Parameter(torch.zeros((out_features, r)))
+            self.lora_A = nn.Parameter(torch.empty((r, in_features)))
+            self.lora_B = nn.Parameter(torch.empty((out_features, r)))
             self.scaling = self.lora_alpha / self.r
             self.reset_parameters()
```

Comment:
I feel like `reset_parameters` should be called automatically by torch during a layer creation. But I highly unconfident about that.

Reply:
It's more of a silent convention. Only FSDP calls it

Reply:
Got it.